docs/source/model_parameters.md

```diff
@@ -0,0 +1,294 @@
+# Simulation Model
+
+## Introduction
+
+Simulation model parameters describe properties of all relevant elements of the observatory including site,
+telescopes, and calibration devices.
+The management, definition, derivation, verification, and validation of the simulation model is central to the functionality of simtools.
+
+Examples for model parameters are the atmospheric models (profile, transmission) or geomagnetic field characteristics for the site,
+description of optical and mechanical properties of the telescope, and detector plan, trigger, and readout settings for the cameras.
+
+For simplicity, the term parameter is used for values, (multi-dimensional) vectors of values
```

Comment:
I would first explain that the parameters are retrieved from the database in form of a dictionary with value, unit, etc. Then I would say that we use the word "parameter" to refer to the value of the parameter

Reply:
Not sure - I think it is important from the start to make clear that we have a relatively 'wide' understanding of what a parameter is. I would prefer to leave it this way.